src/clr-angular/modal/modal.ts

```diff
@@ -67,17 +72,10 @@ export class ClrModal implements OnChanges, OnDestroy {
   constructor(
     private _scrollingService: ScrollingService,
     public commonStrings: ClrCommonStringsService,
+    @Inject(PLATFORM_ID) private platformId: Object,
     @Inject(UNIQUE_ID) public modalId: string
   ) {}
 
-  get sizeClass(): string {
-    if (this.size) {
-      return 'modal-' + this.size;
-    } else {
-      return '';
-    }
-  }
-
```

Comment:
was this something we haven't been using for a while and just getting cleaned up here?

Reply:
yeah. we apparently moved away from interpolating that class string to do a `[class.modal-*]` boolean binding in modal.html